### PR TITLE
Integration test for find financial transaction by Id

### PR DESCRIPTION
**Pull request by zetxdev**

### CODE REVIEW

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    @DisplayName("Should return proper financial transaction when search Id exist in database")
+    @Test
+    public void testGetFinancialTransactionById_whenFindingTransactionWithExistingId_thenReturnFinancialTransactionWithCorrespondingId() throws Exception {
+        Wallet wallet = walletRepository.save(new Wallet("TestWallet"));
+        FinancialTransaction testFinancialTransaction = createTestFinancialTransaction(wallet, "Test1");
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/transactions/{id}", testFinancialTransaction.getId())
+                        .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
```
Comment (DawidStuzynski) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:45]: We need to consider it, but in my opinion this line would be great for debugging only

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    @DisplayName("Should return status NOT_FOUND when search Id does not exist in database")
+    @Test
+    public void testGetFinancialTransactionById_whenSearchIdDoesNotExistInDatabase_thenReturnErrorNotFound() throws Exception {
+        Wallet wallet = walletRepository.save(new Wallet("TestWallet"));
+        FinancialTransaction testFinancialTransaction = createTestFinancialTransaction(wallet, "Test1");
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/transactions/{id}", 999)
+                        .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
```
Comment (DawidStuzynski) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:61]: We need to consider it, but in my opinion this line would be great for debugging only

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java (hunk 1 above)
Comment (zetxdev) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:45]: Indeed, that was for debugging, to see visually what should be returned. It is not critical for the test, so if you want me to remove it I can do it

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java (hunk 2 above)
Comment (zetxdev) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:61]: Indeed, that was for debugging, to see visually what should be returned. It is not critical for the test, so if you want me to remove it I can do it

**Review by dawciobiel — APPROVED** [review]
Looks good to me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java (hunk 2 above)
Comment (odzio33) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/FindTransactionByIDTest.java:61]: I agree with DawidS, please remove this print.

**Review by odzio33 — CHANGES_REQUESTED** [review]
Changes required

**Pull request closed without merging**